.github/workflows/bot-bot.yml

```diff
@@ -45,8 +45,6 @@ jobs:
         if: success() && ! env.CI_SKIP
         run: |
           source cf-scripts/autotick-bot/install_bot_code.sh
-        env:
-          BOT_TOKEN: ${{ secrets.AUTOTICK_BOT_TOKEN }}
```

Comment:
I removed `BOT_TOKEN` for `install_bot_code.sh` entirely everywhere because I could not find where it uses the token.